src/DependencyInjection/Configuration.php

```diff
@@ -721,6 +721,21 @@ private function createCachePluginNode()
                 ->integerNode('default_ttl')
                     ->info('The default max age of a Response')
                 ->end()
+                ->arrayNode('blacklisted_paths')
+                    ->info('An array of regular expression patterns for paths not to be cached. Defaults to an empty array.')
+                    ->defaultValue([])
+                    ->beforeNormalization()
+                        ->castToArray()
+                    ->end()
+                    ->prototype('scalar')
+                        ->validate()
+                            ->ifTrue(function ($v) {
+                                return false === @preg_match('/'.$v.'/', '');
```

Comment:
Do we want to protect with `/`?
Why not use `#` as a delimiter? That is way less likely to be added to the URL.